python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -121,6 +120,19 @@ def from_pretrained(cls,
         optimize_model = kwargs.pop("optimize_model", True)
         user_quantization_config = kwargs.pop("quantization_config", None)
 
+        if user_quantization_config is not None and \
+                "BitsAndBytesConfig" in str(user_quantization_config.__class__):
+            if user_quantization_config.bnb_4bit_quant_type is not None:
+                load_in_low_bit = user_quantization_config.bnb_4bit_quant_type
```

Comment:
you should map original `bnb_4bit_quant_type` to `load_in_low_bit` instead of requiring user to use bigdl low-bit type in this case; if it cannot be mapped, report warning and map to a default value.